android/app/src/main/res/values-es/strings.xml

```diff
@@ -52,11 +52,9 @@
     <string name="disconnect_usb_cable">Desconecte el cable USB o inserte una tarjeta de memoria SD para usar Organic Maps</string>
     <!-- Used in DownloadResources startup screen -->
     <string name="not_enough_free_space_on_sdcard">Libere primero espacio en la memoria SD o almacenamiento USB para usar la aplicación</string>
-    <string name="download_resources">Antes de comenzar, permita que descarguemos en su dispositivo un mapamundi general.
-\nSe requieren %s de almacenamiento.</string>
+    <string name="download_resources">Antes de comenzar, permita que descarguemos en su dispositivo un mapamundi general. \nSe requieren %s de almacenamiento.</string>
```

Comment:
Are line breaks important for this string (i.e. for proper display in the app?)

Reply:
For Android `\n` are important, linebreaks are not.